packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,560 @@
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "LED",
+				"value": 0
+			},
+			{
+				"label": "Switch and LED",
+				"value": 1
+			},
+			{
+				"label": "Pushbutton activated",
+				"value": 2
+			}
+		]
+	},
+	"pushbutton_function_zdb5100": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 4,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Toggle",
+				"value": 0
+			},
+			{
+				"label": "Automatic turn off after the time has expired",
+				"value": 1
+			},
+			{
+				"label": "Automatic turn on after the time has expired",
+				"value": 2
+			},
+			{
+				"label": "Always turn off or dim down",
+				"value": 3
+			},
+			{
+				"label": "Always turn on or dim up",
+				"value": 4
+			}
+		]
+	},
+	"input_funtion_relay": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Function setup of input",
+		"maxValue": 3,
+		"defaultValue": 3,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Level-controlled input",
+				"value": 1
+			},
+			{
+				"label": "Falling-edge controlled input",
+				"value": 2
+			},
+			{
+				"label": "Rising-edge controlled input",
+				"value": 3
+			}
+		]
+	},
+	"output_function_relay": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Output function",
+		"options": [
+			{
+				"label": "Controlled via Z-Wave messages",
+				"value": 0
+			},
+			{
+				"label": "Controlled by input 1",
+				"value": 1
+			}
+		]
+	},
+	"status_led": {
+		"$import": "../../templates/master_template.json#led_indicator_four_options",
+		"label": "Status of LED",
+		"defaultValue": 1,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Enabled",
+				"value": 1
+			},
+			{
+				"label": "Flashes in 1 second interval",
+				"value": 2
+			},
+			{
+				"label": "Flashes in 1/2 second interval",
+				"value": 3
+			}
+		]
+	},
+	"meter_type": {
+		"label": "Meter Type of input",
+		"description": "Meter Type of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 5,
+		"defaultValue": 1,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Input not in use",
+				"value": 0
+			},
+			{
+				"label": "Electricity meter",
+				"value": 1
+			},
+			{
+				"label": "Gas meter.",
+				"value": 2
+			},
+			{
+				"label": "Water meter.",
+				"value": 3
+			},
+			{
+				"label": "Heat meter",
+				"value": 4
+			},
+			{
+				"label": "Cooling meter.",
+				"value": 5
+			}
+		]
+	},
+	"input_scale": {
+		"label": "Scale Type of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 6,
```

Comment:
Any explanation for these? Seems like there should be options.

Reply:
Yes there is, but its unclear to me how to solved it straight out of the box.
this parameter has different states based on another selection?

**Parameter 35, Parameter size 1 byte. Scale Type for input 6.**
Indicates the device type of the currently selected meter type connected to input 6.
This parameter is retained during exclusion / inclusion of the device.
Selected meter type: Value Description
Electricity meter (1) 0 kWh
                                1 kVAh
                                2 W
                                3 Pulse counter (Default)
                                4 V
                                5 A
                                6 Power Factor
Gas meter (2)           0 Cubic meters
                                1 cubic foot
                                2 Reserved
                                3 Impulse counter
Water meter (3)       0 Cubic meters
                                1 cubic foot
                                2 US gallons
                                3 Impulse counter
Heat meter (4)         0 kWh
Cooling meter (5)    0 kWh

Reply:
Hm, okay that's not straightforwards. Maybe add a description referring the user to the manual.

Reply:
but the "description": "Check manual for correct usage", is not applicable is it ?

Reply:
In this case I would argue it is. You can leave a JS comment in the file explaining it, e.g.
```jsonc
// The meaning of this parameter's values change depending on another parameter
```
or so